internal/test_cases/utils.go

```diff
@@ -29,3 +32,21 @@ func createTempFileWithContents(contents string) (string, error) {
 
 	return tmpFile.Name(), nil
 }
+
+func (t *ParseTestCase) logReadableFileContents(logger *logger.Logger) {
```

Comment:
A function on the `ParseTestCase` type shouldn't be in `utils`, it should be in `parse_test_case.go`.

Reply:
This was supposed to be a standalone function, to be used across all test_cases.